hw/vendor/patches/pulp_platform_common_cells/0005-common_cells-Exclude-cb_filter-from-vivado-IP-packag.patch

```diff
@@ -0,0 +1,39 @@
+From 5c93fe9f6f88e23d314ea2244b80485d00e290c9 Mon Sep 17 00:00:00 2001
+From: Noah Huetter <noahhuetter@gmail.com>
+Date: Wed, 23 Feb 2022 13:35:50 +0100
+Subject: [common_cells] Exclude cb_filter from vivado IP packager projects
+
+Signed-off-by: Noah Huetter <noahhuetter@gmail.com>
+---
+ Bender.yml | 8 ++++++--
+ 1 file changed, 6 insertions(+), 2 deletions(-)
+
+diff --git a/Bender.yml b/Bender.yml
+index 1e06d7b0..0631e5c9 100644
+--- a/Bender.yml
++++ b/Bender.yml
+@@ -22,7 +22,9 @@ sources:
+   # etc. Files within a level are ordered alphabetically.
+   # Level 0
+   - src/binary_to_gray.sv
+-  - src/cb_filter_pkg.sv
++  - target: not(all(xilinx,vivado_ipx))
++    files:
++    - src/cb_filter_pkg.sv
```

Comment:
The Vivado IP packager complains about the default parameter assignments in the `cb_filter` module, hence this change

Reply:
:+1: